### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [meganrisi/prj-rev-bwfs-tea-cozy#1]
It's considered best practice to try and always include an [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) on all the images used for a site. The reason for this is because the alt attribute is used for both [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case an image cannot be rendered. 

https://github.com/meganrisi/prj-rev-bwfs-tea-cozy/blob/d74de7630e5087529562d92e785d4252b4e9a43d/tea_cozy/index.html#L13
